src/query-follower/query-follower-types.ts

```diff
@@ -13,113 +20,74 @@ export type QueryFollowerState =
     | "error";
 
 /**
- * Subscribe to the ongoing results of a query,
```

Comment:
Most of this comment is relevant to end-users, not just core developers.

Reply:
Agreed. I've made another attempt to split apart the different things this passage describes to the parts of the class which relates to them.